zdm/tests/test_basic.py

```diff
@@ -17,80 +18,43 @@
 def make_grids():
 
     ############## Initialise parameters ##############
-    params = parameters.init_parameters()
+    state = parameters.State()
 
-    # Test against main
-    params['cosmo'].H0 = 67.74
-    params['cosmo'].Omega_lambda = 0.685
-    params['cosmo'].Omega_m = 0.315
-    params['cosmo'].Omega_b = 0.044
-
-    # Fix Omega_b?
-    if params['cosmo'].fix_Omega_b_h2:
-        params['cosmo'].Omega_b = params['cosmo'].Omega_b_h2/(
-            params['cosmo'].H0/100.)**2
-        print(f"Omega_b adjusted to {params['cosmo'].Omega_b}")
-
-    # Compare to main branch
-    #DEF_Omega_k=0.
-    # dark energy / cosmological constant (in current epoch)
-    #DEF_Omega_lambda=0.685
-    # matter density in current epoch
-    #DEF_Omega_m=0.315 #Plank says 0.315
-    # baryon density
-    #DEF_Omega_b=0.044
-    #DEF_Omega_b_h2=0.0224 #Planck says 0.0224, WMAP 0.02264
-    # hubble constant in current epoch
-    #DEF_H0 = igm.Planck15.H0.value #km s^-1 Mpc^-1 #planck15 used in frb.igm
+    # Variable parameters
+    vparams = {}
+    vparams['cosmo'] = {}
+    vparams['cosmo']['H0'] = 67.74
+    vparams['cosmo']['Omega_lambda'] = 0.685
+    vparams['cosmo']['Omega_m'] = 0.315
+    vparams['cosmo']['Omega_b'] = 0.044
```

Comment:
Question here. Do we have a method to fix either H0 or Omega_b, and automatically set the other one according to Planck? I know Esan is implementing this, not sure if it is contained here or not.

Reply:
yes for `H0`